module/entities/helper.js

```diff
@@ -808,6 +816,11 @@ module.exports = class EntitiesHelper {
                     queryObject["$match"]["_id"]["$in"] = entityIds;
                 }
 
+                if( aclData.length > 0 ) {
```

Comment:
@aman-tunerlabs - Has the same filter added in user extension get profile APIs ?

Reply:
@aks30  for get profile api we just added one extra field called as acl in response.